Fireworks/FWInterface/interface/FWFFLooper.h

```diff
@@ -80,7 +80,7 @@ class FWFFLooper : public CmsShowMainBase, public edm::EDLooperBase
    void requestChanges(const std::string &, const edm::ParameterSet &);
 
    void remakeGeometry(const DisplayGeomRecord& dgRec);
-
+   static const edm::EventSetup* m_setup;
```

Comment:
Actually, the presence of an EDLooper in a config automatically causes the framework to go into single threaded mode.